src/backend.ts

```
@@ -98,329 +41,73 @@ export namespace ResultOfAccessTokenVerify {
-                try {
-                    decodedAccessToken = decodedAccessTokenSchema.parse(decodedAccessToken_original);
-                } catch (error) {
-                    return id<ResultOfAccessTokenVerify.Invalid>({
-                        isValid: false,
-                        errorCase: "does not respect schema",
-                        errorMessage: String(error)
-                    });
+            if (!isSuccess) {
+                switch (errorCause) {
+                    case "missing Authorization header":
+                        assert(false, "29330204");
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)